### PR TITLE
DY madgraphMLM

### PR DESCRIPTION
**Pull request by sihyunjeon**

### CODE REVIEW

**Comment by sihyunjeon** [comment]
Need to wait for the tune

**Comment by sihyunjeon** [comment]
Forgot to add the "how many additional jets in the process" to the dataset names

File: Cards/MadGraph5_aMCatNLO/DY/DYtoLL_0J_madgraphMLM-pythia8/DYtoLL_0J_madgraphMLM-pythia8.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "xqcut": "10",
+        "maxjetflavor": "5"
+    },
+
+    "user":[
+        "set run_card mmll 50",
+        "set run_card auto_ptj_mjj True"
+    ],
+
+    "fragment": [
```
Comment (agrohsje) [inline Cards/MadGraph5_aMCatNLO/DY/DYtoLL_0J_madgraphMLM-pythia8/DYtoLL_0J_madgraphMLM-pythia8.json:16]: Should be change to something like: 
fragment": ["Generator/ExternalLHEProducer.dat","PartonShower/PowhegEmissionVetoPythia8.dat"],
    "fragment_user": [],
    "fragment_vars": {
        "processParameters": [
  ...
  no ?

Comment (sihyunjeon) [reply Cards/MadGraph5_aMCatNLO/DY/DYtoLL_0J_madgraphMLM-pythia8/DYtoLL_0J_madgraphMLM-pythia8.json:16]: closing this, it's a bit outdated

**Pull request closed without merging**